kie-dmn-feel/src/main/java/org/kie/dmn/feel/runtime/functions/ConcatenateFunction.java

```diff
@@ -39,9 +44,10 @@ public List apply(@ParameterName("list") Object[] lists) {
             } else if ( list != null ) {
                 result.add( list );
             } else {
-                return null;
```

Comment:
@etirelli I check in the spec Table 61 at PDF-page 138, printed-page 134 mentions:

> concatenate(list…)
list
return new list that is a
concatenation of the arguments
concatenate([1,2],[3]) = [1,2,3]

Then I notice the test is consistent with this original implementation:
```
{ "concatenate( [1, 2], [3] )", Arrays.asList( BigDecimal.valueOf( 1 ), BigDecimal.valueOf( 2 ), BigDecimal.valueOf( 3 ) ) },
                { "concatenate( [1, 2], 3, [4] )", Arrays.asList( BigDecimal.valueOf( 1 ), BigDecimal.valueOf( 2 ), BigDecimal.valueOf( 3 ), BigDecimal.valueOf( 4 ) ) },
                { "concatenate( [1, 2], null )", null },
```

but I'm not able to understand why, accordingly to the spec, this should be the expected behavior and why not, for example resulting : concatenate( [1, 2], null, 3 ) = [1,2,3] ?

Can you kindly help me in clarifying this please?
I think I've seen this pattern elsewhere and I'd like to be sure. Thx!